node/node.go

```diff
@@ -1235,6 +670,22 @@ func (n *Node) startPrometheusServer(addr string) *http.Server {
 	return srv
 }
 
+// starts a ppro
+func (n *Node) startPprofServer() *http.Server {
+	srv := &http.Server{
+		Addr:              n.config.RPC.PprofListenAddress,
+		Handler:           nil,
```

Comment:
Why is the handler `nil`?

Reply:
Interesting. It looks like the `pprof` library import was removed here, so I'm not sure `pprof` is being linked in at all in this code.

Reply:
Ah lol I removed that because `gosec` said: "Profiling endpoint is automatically exposed on /debug/pprof"

Reply:
I added a test (`TestPprofServer`) that seems to be passing